lib/galaxy/datatypes/isa.py

```diff
@@ -0,0 +1,520 @@
+"""
+ISA datatype
+
+See http://isa-tools.org
```

Comment:
This url seems down to me (and my colleague @natefoo), redirects to a non existing page at `http://apps1047.gold-traffic94.loan/?utm_medium=NQ3aDvyuBCtafRQJPeFC66tm%2bMNW8T%2baflxP0d0AJGo%3d&t=main4`

Not sure what is up with that.

Reply:
The website address is correct. Except the site has been hacked. I will replace the URL with the GiHub URL for the organisation ISA tools.